src/exception/call_stack.cr

```diff
@@ -19,6 +19,37 @@ end
 
 # :nodoc:
 struct Exception::CallStack
+  record Frame,
+    file : String?,
+    line : Int32?,
+    column : Int32?,
+    name : String?,
+    sname : String?,
```

Comment:
What's `sname`?

Reply:
It comes from `LibC::DlInfo#dli_sname`.

Reply:
Let's not expose this

Reply:
It helps with clean API and I don't see a reason to _not to_ expose it.